activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/Process.java

```diff
@@ -274,6 +276,22 @@ public void setCandidateStarterGroups(List<String> candidateStarterGroups) {
     this.candidateStarterGroups = candidateStarterGroups;
   }
 
+  public boolean isCandidateStarterUsersExists() {
+    return candidateStarterUsersExists;
+  }
```

Comment:
Not sure how will that sound in case of `activiti:candidateStarterUsers="" activiti:candidateStarterGroups=""` because in that case the value should be set to true.

Reply:
Yes, in this case it's defined to empty, a bit controversial maybe.